src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePresentation.java

```diff
@@ -354,13 +354,11 @@ private static Object getValueAtDetailAST(DetailAST ast, int column) {
      * @return DetailNode tree
      */
     private DetailNode getJavadocTree(DetailAST blockComment) {
-        DetailNode javadocTree = blockCommentToJavadocTree.get(blockComment);
-        if (javadocTree == null) {
-            javadocTree = new JavadocDetailNodeParser().parseJavadocAsDetailNode(blockComment)
+        return blockCommentToJavadocTree.computeIfAbsent(blockComment, blockCommentNode -> {
```

Comment:
It is possible to extract the lambda body to a helper method to avoid creation of a helper class:
```java
    /**
     * Gets Javadoc (DetailNode) tree of specified block comments.
     * @param blockComment Javadoc comment as a block comment
     * @return DetailNode tree
     */
    private DetailNode getJavadocTree(DetailAST blockComment) {
        return blockCommentToJavadocTree.computeIfAbsent(blockComment,
                ParseTreeTablePresentation::parseJavadocTree);
    }

    /**
     * Parses Javadoc (DetailNode) tree of specified block comments.
     * @param blockComment Javadoc comment as a block comment
     * @return DetailNode tree
     */
    private static DetailNode parseJavadocTree(DetailAST blockComment) {
        return new JavadocDetailNodeParser().parseJavadocAsDetailNode(blockComment)
                .getTree();
    }
```
then the method `getJavadocTree` can be inlined (it has one usage in `getChildAtDetailAst`)

Reply:
I like you approach .... but not for extra call creation  but for more readability as it resolved not good part `blockComment, blockCommentNode ->`.

fixed.

Reply:
@pbludov , it is better to be more exact in review process.
just making comments is good when you approve but would like to discuss somehting, while second reviewer can proceed in review.

if you would like to discuss something and wait for response ... it is better to make "Request Changes" to make it clear to want point to be finished before approval.